shared/src/commonMain/kotlin/com/example/myplants/plants/domain/Plant.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.myplants.plants.domain
+
+data class Plant(
+    val id: String,
+    val name: String,
+    val description: String,
+    val waterAmount: String,
+    val waterDays: List<String>,
```

Comment:
It's a bit unclear what a String represents here, I would rather work with `ZonedDateTime`